students/john_hunter/lesson03/mailroom.py

```diff
@@ -0,0 +1,227 @@
+#----------------------------------------#
+#Python 210
+# Assignment01: mailroom.py
+# Desc: Allows a user to read, and update a list of donors and thier donations.
+#       Allows the user to generate a thank you email or report on historical 
+#       summaries of the donors activities. 
+# Change Log:
+# Johnh, 2020-Sept-22 Script Created 
+# Johnh, 2020-Sept-22 Header and basic functions defined  
+# Johnh, 2020-Sept-22 Some previous text UI work copied from other script
+# Johnh, 2020-Sept-24 Basic clean up to get the script to execute, added main
+# Johnh, 2020-Sept-26 Definition of the send thank you function
+# Johnh, 2020-Oct-21 Code satsfies all requirements 
+# Johnh, 2020-Feb-25 Readablity Review, comments cleanup, SoC formatting
+# Johnh, 2020-Feb-25 Tested, passed
+# Johnh, 2020-Feb-25 Added to Git and submitted
+#----------------------------------------#
+
+import time
+import sys
+
+
+## Donors in the global namespace, dictionary by donor ID as key, first name
+# last name and donation history as additional data entries 
+## donor 
+donors =	{ '001' : ['Merriweather', 'Frank',10,15,100], '002' : ['Tran', 'Thomas',5,17,23], 
+              '003' : ['Terrance', 'Stephanie', 31, 48, 108], '004': ['Robidas', 'Sam', 4,90, 101],
+              '005' : ['Cohen', 'Sandy', 29, 41, 70], '006' : ['Kemp', 'Shioban', 2, 23000, 19]
+}
```

Comment:
in this lesson you should be using a list for your main data structure for donors.

see guidelines for help: https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/mailroom_tutorial.html#exercise-mailroom-part1-tutorial